### PR TITLE
Add parsers for common measurement matching any test name

### PR DESCRIPTION
**Pull request by mm4tt**

### CODE REVIEW

File: perfdash/config.go [hunk 1]
```diff
@@ -397,6 +437,10 @@ var (
 				OutputFilePrefix: "SystemPodMetrics",
 				Parser:           parseSystemPodMetrics,
 			}},
+			"SystemPodMetrics": []TestDescription{{
```
Comment (jkaniuk) [inline perfdash/config.go:440]: Do we really need this kind of repetition?
It is stating for the third time almost the same thing.
There was some rule of the thumb - up to two occurrences - don't bother, three and up - deduplicate.

Comment (mm4tt) [reply perfdash/config.go:440]: Yeah, but in this case I don't think that work on refactoring will ever pay off, this third repetition that I added is the "catch-all" clause, we rather won't need another one. Also, this whole code should be refactored, but we made the decision to not invest into perf-dash more than absolutely necessary.

**Pull request merged**